configuration/pih/pih-config-haiti-hsn.json

```diff
@@ -35,8 +37,8 @@
   },
 
   "locationTags": {
-    "Lab Results Location": ["COVID-19 | Izolman | HSN", "COVID-19 | UMI | HSN"],
-    "Login Location": ["COVID-19 | Izolman | HSN", "COVID-19 | UMI | HSN", "Sante Mantal | HSN"],
```

Comment:
@jmbabazi  we don't want to change the tags here.  HSN currently as 3 "sub-locations" (see attached image).

We should decide if these are still relevant and if we still want sub-locations at HSN... if so, we may need to add new locations for NCD and OB-GYN?

![2021-08-31_17-38](https://user-images.githubusercontent.com/1163015/131580334-61188c53-e8b6-4a8b-8f07-10d09cfe4733.png)

Reply:
Do we want to remove COVID from HSN?  Maybe keep that around although they haven't been using...